docs/reference/modules/indices/query_cache.asciidoc

```diff
@@ -1,27 +1,35 @@
 [[query-cache]]
 === Node Query Cache
 
-The query cache is responsible for caching the results of queries.
-There is one queries cache per node that is shared by all shards.
-The cache implements an LRU eviction policy: when a cache becomes full, the
-least recently used data is evicted to make way for new data.
-It is not possible to look at the contents being cached.
+The query cache is responsible for caching the results of queries. There is one 
+queries cache per node that is shared by all shards. The cache implements an LRU 
+eviction policy: when a cache becomes full, the least recently used data is 
+evicted to make way for new data. It is not possible to look at the contents 
+being cached.
 
```

Comment:
I'd take the opportunity to tighten this up a bit. 

The results of queries used in the filter context are cached in the node query cache for fast lookup.
The cache uses an LRU eviction policy: when the cache is full, the least recently used query results are evicted to make way for new data. You cannot inspect the contents of the query cache.